src/Lean/Meta/Tactic/TryThis.lean

```diff
@@ -426,7 +426,7 @@ def addSuggestions (ref : Syntax) (suggestions : Array Suggestion)
     (codeActionPrefix? : Option String := none) : MetaM Unit := do
   if suggestions.isEmpty then throwErrorAt ref "no suggestions available"
   let msgs := suggestions.map toMessageData
-  let msgs := msgs.foldl (init := MessageData.nil) (fun msg m => msg ++ m!"\n• " ++ m)
+  let msgs := msgs.foldl (init := MessageData.nil) (fun msg m => msg ++ m!"\n• " ++ .nest 2 m)
   logInfoAt ref m!"{header}{msgs}"
   addSuggestionCore ref suggestions header (isInline := false) origSpan? style? codeActionPrefix?
```

Comment:
@nomeata: this line is the one that produces the widget, and it doesn't use `msgs` at all.

In fact I think the rendering of the widget is subtly wrong in an unrelated way, but arguably that's out of scope for this PR.